web/src/components/member/MemberBadge/MemberAvatar.tsx

```
@@ -0,0 +1,83 @@
+  const [path] = getAccountGetAvatarKey(id);
+
+  return `${API_ADDRESS}/api${path}`;
+}
+
+export function avatarSize(size: Props["size"]) {
+  switch (size) {
+    case "sm":
+      return { width: 24, height: 24 };
+    case "md":
+      return { width: 36, height: 36 };
+    case "lg":
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)